main.go

```diff
@@ -141,7 +141,22 @@ func main() {
 		if err != nil {
 			setupLog.Error(err, "cannot start http server")
 		}
-		address := fmt.Sprintf(":%v", Port)
+		var cm v1.ConfigMap
+		cmKey := client.ObjectKey{Namespace: "k4e-operator-system", Name: "k4e-operator-manager-config"}
```

Comment:
What do you think about using the config map as env variables source in the deployment, like here: https://github.com/openshift/assisted-service/blob/04326d154f72cb71ece287dad19b7f016464ae88/deploy/assisted-service.yaml#L40?

Reply:
I wonder whether it would be possible to parametrize namespace.

Reply:
with Jakub's suggestion it won't be needed